HIP-Basic/assembly_to_executable/assembly_to_executable_vs2019.vcxproj.filters

```diff
@@ -0,0 +1,53 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
For file names and folders, are using -, _, or CamelCase or camelCase? Let's add it in a style guide and stick with it.

Reply:
Directories use pascal case in general, with the exception of those in Libraries, which should follow the same casing style as the library they represent. Files and directories that represent an example use snake case. We've added this to the guide in Docs/, and re-named the directories that did not follow it accordingly.